tools/merge-volume-into-data-layer/main.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+
+from argparse import ArgumentParser
+import zipfile
+import shutil
+import os
+import sys
+import random
+import string
+import wkw
+import numpy as np
+import re
+from operator import add
+
+def main():
+    args = create_parser().parse_args()
+    confirmation_prompt(args)
+    tracing_tmpdir_path = extract_tracing_zip(args)
+
+    tracing_dataset = wkw.Dataset.open(os.path.join(tracing_tmpdir_path, '1'))
+    segmentation_dataset = wkw.Dataset.open(os.path.join(args.segmentation_path))
```

Comment:
Maybe add `1` to segmentation path here as well (and don't require the user to provide it). This prevents users to accidentally merge data from different zoom levels.